.github/actions/bootstrap/action.yaml

```diff
@@ -5,9 +5,56 @@ inputs:
     description: "Platform name"
     required: false
     default: none
+  bootstrap-log-name:
+    description: "Bootstrap log name"
+    required: false
+    default: bootstrap-logs-${{ github.job }}
+outputs:
+  cache-hit:
+    description: "Bootstrap environment was restored from cache"
+    value: ${{ fromJSON(steps.bootstrap-cache.outputs.outputs).cache-hit }} # retry returns all outputs in `outputs`
+
 runs:
   using: "composite"
   steps:
-    - name: Bootstrap
+    - name: Calculate bootstrap cache key
```

Comment:
Block auto-merge so tests can be run a couple of extra times to ensure this is stable.

Reply:
I did a few re-runs of various jobs and all attempts passed.